lang/cs/Org.Apache.REEF.Driver.Tests/Org.Apache.REEF.Driver.Tests.csproj

```diff
@@ -0,0 +1,70 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is for driver unit testing. We usually have unit tests for each project if needed. REEF.Tests is for E2E testing. 

Currently there is not unit test project for driver.